config.toml.example

```diff
@@ -171,37 +176,37 @@ changelog-seen = 2
 # first compiler.
 #
 # Defaults to host platform
-#build = "x86_64-unknown-linux-gnu"
+#build = <host platform>
```

Comment:
I think saying host platform is a plausibly a bit confusing here given host key below, though it's always been a bit hard to pin down what build really is.